fizzbuzz.js

```diff
@@ -0,0 +1,26 @@
+  const fizzbuzz = (number) => {
+    if ((number % 3 === 0) && (number % 5 === 0)) {
+        return 'fizzbuzz';
+    } else if (number % 3 === 0) {
+        return 'fizz';
+    } else if (number % 5 === 0) {
+        return 'buzz';
+    } else {
+        return number;
+    }
+  };
+      
+  const main = () => {
+    for (var i = 1; i <= 100; i++) {
+      console.log(fizzbuzz(i));
+    }
+  }
+  main();
+
+  const test = () => {
+    const array = [4, 15, 20, 21];
+    for(var i = 0; i <= 3; i++) {  
```

Comment:
Maybe try [for ... of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) instead of a traditional for loop.